Python/pylifecycle.c

```diff
@@ -1724,7 +1726,8 @@ finalize_interp_clear(PyThreadState *tstate)
         _PyArg_Fini();
         _Py_ClearFileSystemEncoding();
     }
-
+    /* Finalize deepfrozen code objects */
+    _Py_Deepfreeze_Fini();
```

Comment:
Is this the right place to dealloc the frozen objects? Shouldn't it be in the ``is_main_interp`` block?

The function name is self-explaining. The comment does not add any additional information.

```suggestion
    _Py_Deepfreeze_Fini();
```